### DISCUSSION TITLE
Frigate & motion

### DISCUSSION THREAD

**Discussion by Trzinka** [blakeblackshear/frigate#9557] [General]
# Hi!

**I had a discussion: https://github.com/blakeblackshear/frigate/discussions/9465 how to show the camera in the lovelace card when it is active and I came to the conclusion with the interlocutor that at that moment it was only possible via motion. Everything worked as expected as it was set up.**

`After the upgrade, something has obviously changed, as motion no longer shows motion detection as it did before:`

![Frigate-Motion](https://github.com/blakeblackshear/frigate/assets/40424965/86eb1623-1474-4c0f-aa33-8a97ef2dc0fb)

# Is there maybe some new/changed setting in Frigate that I've missed?

**Comment by NickM-27** [comment]
Like the release notes said, motion detection itself has changed. But the way motion is reported to home assistant has not changed.

**Reply by Trzinka** [reply]
# Hi!

**Thanks for the reply.**

`Sorry I wasn't paying enough attention to notice.`
![Frigat-Change](https://github.com/blakeblackshear/frigate/assets/40424965/a8da7d98-8134-45c1-8e7c-3060e04f0b36)

`But now I'm completely confused because I don't know which entity to follow now?`
![Frigate-New](https://github.com/blakeblackshear/frigate/assets/40424965/a1395971-8b10-4c3e-b0ba-29fd9e07ab63)


> An object that is not moving is detected on the camera called Jedilnica!
> On the camera named Diane, there are 2 people moving and the car standing still.

# Which entity should I follow?

**Reply by NickM-27** [reply]
the entities in the integration have not changed, it's not clear what you're looking for

**Reply by Trzinka** [reply]
# I have no idea what this is supposed to represent! It may or may not help you!

![Frigate MQTT](https://github.com/blakeblackshear/frigate/assets/40424965/3c152067-730c-40ae-ba20-660b1d64eac2)

**Reply by Trzinka** [reply]
## Do you remember ours discussion: https://github.com/blakeblackshear/frigate/discussions/9465

`I called`
```yaml
              - type: conditional
                conditions:
                  - condition: state
                    entity: binary_sensor.jedilnica_motion
                    state: 'on'
```
`because there was no other solution!`

# It worked until the upgrade, but now it doesn't!

**Reply by NickM-27** [reply]
> It worked until the upgrade, but now it doesn't!

like I said, nothing about the motion entity has changed. it is not clear what is not working

**Reply by Trzinka** [reply]
# Apparently something has changed!

![Frigate history](https://github.com/blakeblackshear/frigate/assets/40424965/7e7b9f8f-8f4f-49fa-896e-8f45638d61d3)

**Reply by NickM-27** [reply]
motion detection itself has changed, like it says in the release notes. The docs explain how to tune motion detection https://docs.frigate.video/configuration/motion_detection

**Reply by Trzinka** [reply]
# So I need to enable motion in frigate.yaml?

`Something like this:`
```yaml
motion:
  threshold: 13
  contour_area: 30
  frame_height: 50
  improve_contrast: False
```

`this alone is not enough:`
![Frigate snapshot](https://github.com/blakeblackshear/frigate/assets/40424965/6716805a-a663-4a14-b00d-9dd6747c7340)
?

**Reply by Trzinka** [reply]
# Somehow it started working.

**Obviously, I will have to read all the instructions from the beginning.**

# Thank you.
